governance/votes/Executive Vote - May 16, 2024.md

```diff
@@ -0,0 +1,131 @@
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+- A total of TBD MKR will be distributed to TBD Aligned Delegates as compensation for May 2024.
+- 20.84 MKR will be distributed as part of a whistleblower bounty.
+- Multiple stability fees will be reduced, as will the Dai Savings Rate.
+- A keeper network job will be updated.
+- 5,358,007 DAI and 1,969.17 MKR will be transferred to the Launch Project.
+- A total of 55,000 DAI will be distributed as part of a Bug Bounty payment.
+- A Spark proxy spell will be triggered.
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **30 hours**.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+
+---
+
+## Proposal Details
+
+### Delegate Compensation for May 2024
+
+$executive_entry_description_1
+
+### Whistleblower Bounty
+
+* **Authorization:** [Atlas 2.6.6](https://mips.makerdao.com/mips/details/MIP101#2-6-6-aligned-delegate-operational-security)
+* **Proposal:** [Forum post](https://forum.makerdao.com/t/ad-derecognition-due-to-operational-security-breach-april-5-2024/24043)
+
+As a bounty for the information leading to the derecognition of TrueName delegate, the following transfer will be made, if this executive proposal passes:
+
+- Transfer **20.84 MKR** to forum user Compacter at [0xbbd4bC3FE72691663c6ffE984Bcdb6C6E6b3a8Dd](https://etherscan.io/address/0xbbd4bC3FE72691663c6ffE984Bcdb6C6E6b3a8Dd).
+
+### Parameter Updates
+
+#### [Stability Fee (SF)](https://mips.makerdao.com/mips/details/MIP104#14-3-1-3-stability-fee-sf-) Changes
+
+* **Authorization:** [Forum post](https://forum.makerdao.com/t/stability-scope-parameter-changes-13-under-sta-article-3-3/24250/2)
+* **Proposal:** [Forum post](https://forum.makerdao.com/t/stability-scope-parameter-changes-13-under-sta-article-3-3/24250)
+
+If this executive proposal passes, as per the recommendation of the Stability Scope Advisory Council Member, BA Labs, the following SFs will be changed:
+
+- Decrease the ETH-A Stability Fee by 2 percentage points from 10.25% to **8.25%**.
+- Decrease the ETH-B Stability Fee by 2 percentage points from 10.75% to **8.75%**.
+- Decrease the ETH-C Stability Fee by 2 percentage points from 10.00% to **8.00%**.
+- Decrease the WSTETH-A Stability Fee by 2 percentage points from 11.25% to **9.25%**.
```

Comment:
And so on

Reply:
not really necessary in our opinion so we won't be changing these